specs/core/0_beacon-chain.md

```diff
@@ -2102,16 +2102,21 @@ def update_validator_registry(state: BeaconState) -> None:
             activate_validator(state, index, is_genesis=False)
 
     # Exit validators within the allowable balance churn
-    balance_churn = 0
-    for index, validator in enumerate(state.validator_registry):
-        if validator.exit_epoch == FAR_FUTURE_EPOCH and validator.initiated_exit:
-            # Check the balance churn would be within the allowance
-            balance_churn += get_effective_balance(state, index)
-            if balance_churn > max_balance_churn:
-                break
+    if state.current_epoch < state.validator_registry_update_epoch + LATEST_SLASHED_EXIT_LENGTH:
```

Comment:
Why are you blocking exits if there have been `LATEST_SLASHED_EXIT_LENGTH` epochs since last registry update?

Reply:
See discussing [here](https://github.com/ethereum/eth2.0-specs/pull/785#discussion_r266187726). It's a simplification to avoid caching a `slashed_balance`.